wrangler/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.0",
   "main": "./src/index.js",
   "scripts": {
+    "preinit": "[ \"${CI}\" = true ] && npx envdist -e || npx envdist",
+    "init": "export $(cat .env|tr -d \"[:blank:]\"|xargs -d '\n') && envsubst < wrangler.toml.dist > wrangler.toml",
```

Comment:
Read all variables defined in `.env` (that was written in pre-init) as ENV-vars into current (sub-)shell before filling `wrangler.toml` file from template with variable values. This may seem like a duplicate in GH action-case but actually it is not, because in `.env.dist` default values can be defined.